OpenSim/Actuators/DeGrooteFregly2016Muscle.cpp

```diff
@@ -279,9 +299,8 @@ void DeGrooteFregly2016Muscle::calcMuscleLengthInfoHelper(
 void DeGrooteFregly2016Muscle::calcFiberVelocityInfoHelper(
         const SimTK::Real& muscleTendonVelocity, const SimTK::Real& activation,
         const bool& ignoreTendonCompliance,
-        const bool& isTendonDynamicsExplicit,
-        const MuscleLengthInfo& mli, FiberVelocityInfo& fvi,
-        const SimTK::Real& normTendonForce = SimTK::NaN,
+        const bool& isTendonDynamicsExplicit, const MuscleLengthInfo& mli,
```

Comment:
Neither a review, nor blocking, nor do you have to answer, but why is it that devs default to using `const T&` for primitive value types like `bool` or `SimTK::Real`?

Reply:
No good reason. We probably should pass such primitive types by value.